src/js/options.js

```diff
@@ -652,17 +664,26 @@ function toggleWebRTCIPProtection() {
   });
 }
 
+/**
+ * Update the user preferences displayed for this origin.
+ * These UI changes will later be used to update user preferences data.
+ * @param event {Event} Click event triggered by user.
+ */
+//TODO unduplicate this code? since it's also in popup
 function updateOrigin(event) {
+  // get the origin and new action for it
   var $elm = $('label[for="' + event.currentTarget.id + '"]');
   log('updating origin for', $elm);
-  var $switchContainer = $elm.parents('.switch-container').first();
-  var $clicker = $elm.parents('.clicker').first();
   var action = $elm.data('action');
+
+  // replace the old action with the new one
+  var $switchContainer = $elm.parents('.switch-container').first();
   $switchContainer.removeClass([
     constants.BLOCK,
     constants.COOKIEBLOCK,
     constants.ALLOW,
     constants.NO_TRACKING].join(" ")).addClass(action);
+  var $clicker = $elm.parents('.clicker').first();
   htmlUtils.toggleBlockedStatus($($clicker), action);
```

Comment:
I think this is redundant (make a jQuery object out of a jQuery object) and a noop. Looks like a production code nit that exists in a couple of places at least (here and in the popup).